custom_components/powercalc/data/signify/LLC012/model.json

```
@@ -4,6 +4,10 @@
     "name": "Hue Bloom Table Lamp",
     "standby_power": 0.3,
     "calculation_strategy": "lut",
-    "aliases": []
+    "aliases": [
+        "LLC011",
+        "LLC013",
+        "7299760PH"
```

```suggestion
        "7299760PH",
        "8718291488668",
        "915004339401"
```

As found here https://zigbee.blakadder.com/Philips_LLC012.html.